app/adapters/application.js

```diff
@@ -1,13 +1,27 @@
 import DS from 'ember-data';
 import config from '../config/environment';
+import { inject as service } from '@ember/service';
+import { computed } from '@ember/object';
 
 const { errorsHashToArray, RESTAdapter } = DS;
 
 export default RESTAdapter.extend({
+  session: service(),
+
   host: config.API.host,
 
   namespace: 'api',
 
+  headers: computed('session.token', function() {
```

Comment:
Was this not working before? I remember we used simple auth for this, I think.

Reply:
`ember-simple-auth` isn't being used in the app so I had to add it myself.